<!doctype html>
<html lang="en">
    
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

        <title>Nils Eling</title>
        <meta name="description" content="An interactive getting started guide for Brackets.">
        <meta name="author" content="Nils Eling">
        <link rel="stylesheet" href="css/bootstrap.min.css">
        <link rel="stylesheet" href="css/b">
        <link rel="stylesheet" href="css/custom.css">
        <link rel="stylesheet" href="css/styles.css">
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons/css/academicons.min.css">
        <script src="https://kit.fontawesome.com/ca4e388e9b.js" crossorigin="anonymous"></script>
        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
    </head>

    <body>
        
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="index.html">
                <span class="d-block d-lg-none">Nils Eling</span>
            </a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav">
                    <li class="nav-item">
                     <a href="https://github.com/nilseling" target="_blank" style="color:#dee2e6;">
                        <i class="fab fa-github fa-2x"></i>
                    </a>
                    <a href="https://twitter.com/NilsEling" target="_blank" style="color:#dee2e6;">
                        <i class="fab fa-twitter fa-2x"></i>
                    </a>
                    <a href="https://www.linkedin.com/in/nils-eling-a3992518a/" target="_blank" style="color:#dee2e6;">
                        <i class="fab fa-linkedin fa-2x"></i>
                    </a>
                    <a href="https://scholar.google.com/citations?user=kBIvrFoAAAAJ&hl=en" target="_blank" style="color:#dee2e6;">
                        <i class="ai ai-google-scholar ai-2x"></i>
                    </a>
                    <a href="https://hub.docker.com/u/nilseling" target="_blank" style="color:#dee2e6;">
                        <i class="fab fa-docker fa-2x"></i>
                    </a>
                    </li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#cytomapper">cytomapper</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#basics">BASiCS</a></li>
                </ul>
            </div>
        </nav>
        
        <div class="container-fluid" id="navBar2">
            <nav class="navbar navbar-expand-lg navbar-light bg-secondary fixed-top" id="topNav">
                <a class="navbar-brand" href="index.html">Nils Eling</a>
                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#largeNavbar" aria-controls="largeNavbar" aria-expanded="false" aria-label="Toggle navigation">
                    <span class="navbar-toggler-icon"></span>
                </button>
                <div class="collapse navbar-collapse" id="largeNavbar">
                    <ul class="navbar-nav">
                        <li class="nav-item"><a class="nav-link" href="index.html">About</a></li>
                        <li class="nav-item"><a class="nav-link" href="research.html">Research</a></li>
                        <li class="nav-item"><a class="nav-link active" href="software.html">Software</a></li>
                        <li class="nav-item"><a class="nav-link" href="publications.html">Publications</a></li>
                        <li class="nav-item"><a class="nav-link" href="cv.html">CV</a></li>
                    </ul>
                </div>
            </nav>
        </div>
        
    <div class='my-container' id="cytomapper" style="background-color: #FFFFFF">
        <div>
            <p style="font-size: 40px; color: black"><samp>cytomapper</samp></p>
            <div class="row">
                <div class="col">
                    <p style="font-size: 20px;">Highly multiplexed imaging cytometry acquires the single-cell expression of selected proteins in a spatially-resolved fashion. These measurements can be visualized across multiple length-scales. First, pixel-level intensities represent the spatial distributions of feature expression with highest resolution. Second, after segmentation, expression values or cell-level metadata (e.g. cell-type information) can be visualized on segmented cell areas. This package contains functions for the visualization of multiplexed read-outs and cell-level information obtained by multiplexed imaging cytometry. The main functions of this package allow (i) the visualization of pixel-level information across multiple channels, (ii) the display of cell-level information (expression and/or metadata) on segmentation masks and (iii) the interactive gating and visualization of cells.</p>
                    <p>Bioconductor: <a href="https://www.bioconductor.org/packages/release/bioc/html/cytomapper.html">https://www.bioconductor.org/packages/release/bioc/html/cytomapper.html</a></p>
                    <p>Github: <a href="https://github.com/BodenmillerGroup/cytomapper">https://github.com/BodenmillerGroup/cytomapper</a></p>
                    <p>Docs: <a href="https://bodenmillergroup.github.io/cytomapper/">https://bodenmillergroup.github.io/cytomapper/</a></p>
                    <p>Paper: <a href="https://www.biorxiv.org/content/10.1101/2020.09.08.287516v1">cytomapper: an R/Bioconductor package for visualisation of highly multiplexed imaging data</a></p>
                </div>
                <div class="col-3">
                    <span><img class="img-fluid" src="img/cytomapper_sticker.png" alt="cytomapper" style="width:200px; mix-blend-mode: multiply; position: absolute; top: 50%; -ms-transform: translateY(-50%); transform: translateY(-50%);"/></span>
                </div>
            </div>
        </div>
    </div>
        
    <div class='my-container' id="basics" style="background-color: #EDFBFF">
        <div>
            <p style="font-size: 40px; color: black"><samp>BASiCS</samp></p>
            <div class="row">
                <div class="col">
                    <p style="font-size: 20px;">Single-cell mRNA sequencing can uncover novel cell-to-cell heterogeneity in gene expression levels in seemingly homogeneous populations of cells. However, these experiments are prone to high levels of technical noise, creating new challenges for identifying genes that show genuine heterogeneous expression within the population of cells under study. BASiCS (Bayesian Analysis of Single-Cell Sequencing data) is an integrated Bayesian hierarchical model to perform statistical analyses of single-cell RNA sequencing datasets in the context of supervised experiments (where the groups of cells of interest are known a priori, e.g. experimental conditions or cell types). BASiCS performs built-in data normalisation (global scaling) and technical noise quantification (based on spike-in genes). BASiCS provides an intuitive detection criterion for highly (or lowly) variable genes within a single group of cells. Additionally, BASiCS can compare gene expression patterns between two or more pre-specified groups of cells. Unlike traditional differential expression tools, BASiCS quantifies changes in expression that lie beyond comparisons of means, also allowing the study of changes in cell-to-cell heterogeneity. The latter can be quantified via a biological over-dispersion parameter that measures the excess of variability that is observed with respect to Poisson sampling noise, after normalisation and technical noise removal. Due to the strong mean/over-dispersion confounding that is typically observed for scRNA-seq datasets, BASiCS also tests for changes in residual over-dispersion, defined by residual values with respect to a global mean/over-dispersion trend.</p>
                    <p>Bioconductor: <a href="https://www.bioconductor.org/packages/release/bioc/html/BASiCS.html">https://www.bioconductor.org/packages/release/bioc/html/BASiCS.html</a></p>
                    <p>Github: <a href="https://github.com/catavallejos/BASiCS">https://github.com/catavallejos/BASiCS</a></p>
                    <p>Docs: <a href="https://bioconductor.org/packages/release/bioc/vignettes/BASiCS/inst/doc/BASiCS.html">https://bioconductor.org/packages/release/bioc/vignettes/BASiCS/inst/doc/BASiCS.html</a></p>
                    <p>Paper: <a href="https://www.sciencedirect.com/science/article/pii/S2405471218302783">Correcting the Mean-Variance Dependency for Differential Variability Testing Using Single-Cell RNA Sequencing Data</a></p>
                </div>
                <div class="col-3">
                    <span><img class="img-fluid" src="img/BASiCS.png" alt="BASiCS" style="width:200px; mix-blend-mode: multiply; position: absolute; top: 50%; -ms-transform: translateY(-50%); transform: translateY(-50%);"/></span>
                </div>
            </div>
        </div>

    </div>
        
    </body>

</html>